mosquitto/data/run.sh

```diff
@@ -91,7 +95,7 @@ else
 fi
 
 # Enable SSL if exists configs
-if [ -e "/ssl/$CERTFILE" ] && [ -e "/ssl/$KEYFILE" ]; then
+if [ -e "/ssl/$CAFILE" ] && [ -e "/ssl/$CERTFILE" ] && [ -e "/ssl/$KEYFILE" ]; then
```

Comment:
Well, that work only now if you have also a ca file

Reply:
It is already the case with cafile being set to certfile currently assuming certfile contains the ca (fullchain).

Reply:
thats true. cafile is already required, but just hard-coded to point to the same place as certifle. You could always just make the default: "cafile": "certfile.pem" along with "require_certificate": false. This way it will continue to work by default but is configurable if desired.

Reply:
OK yes I see what you mean, I had several iterations before this so my previous answer is wrong and you are both correct 😄
I'm not sure what to do here:
1. Have self-explained and accurate default config but not backward compatible which makes it easy for new users (I think) and forces old users to update.
2. Have default config backward compatible but more misleading for new users (cafile and certfile both pointing to fullchain.pem, not even sure that actually works)

I think I went for 2 but had second thoughts and switched to 1 😆

Reply:
To my own defense, I did say in my previous comment that I made that switch 😌 

> [...] All that comes with non-breaking defaults although for people having already configured the SSL it might not be backward compatible. [...]